qiita_pet/static/js/networkVue.js

```diff
@@ -1120,7 +1120,11 @@ Vue.component('processing-graph', {
     show_loading('processing-network-div');
     $("#processing-network-div").hide();
 
-    $('#run-btn').on('click', function() { vm.runWorkflow(); });
+    $('#run-btn').on('click', function() {
+      $('#run-btn').attr('disabled', true);
```

Comment:
If there's an error with the submission request, then runWorkflow is going to catch that error in the `fail` method, right (line 351)? If so, can you remove the disabled class and reset the innerHTML for this button there?